src/MusicalScore/ScoreIO/InstrumentReader.ts

```diff
@@ -419,91 +418,14 @@ export class InstrumentReader {
-          for (const staffDetailsNode of staffDetailsNodes) {
-            const staffLinesNode: IXmlElement = staffDetailsNode.element("staff-lines");
-            if (staffLinesNode) {
-              let staffNumber: number = 1;
-              const staffNumberAttr: Attr = staffDetailsNode.attribute("number");
-              if (staffNumberAttr) {
-                staffNumber = parseInt(staffNumberAttr.value, 10);
-              }
-              this.instrument.Staves[staffNumber - 1].StafflineCount = parseInt(staffLinesNode.value, 10);
-            }
-          }
-          // check multi measure rest
-          const measureStyle: IXmlElement = xmlNode.element("measure-style");
-          if (measureStyle) {
-            const multipleRest: IXmlElement = measureStyle.element("multiple-rest");
-            if (multipleRest) {
-              // TODO: save multirest per staff info a dictionary, to display a partial multirest if multirest values across staffs differ.
-              //   this makes the code bulkier though, and for now we only draw multirests if the staffs have the same multirest lengths.
-              // if (!currentMeasure.multipleRestMeasuresPerStaff) {
-              //   currentMeasure.multipleRestMeasuresPerStaff = new Dictionary<number, number>();
-              // }
-              const multipleRestValueXml: string = multipleRest.value;
-              let multipleRestNumber: number = 0;
-              try {
-                multipleRestNumber = Number.parseInt(multipleRestValueXml, 10);
-                if (currentMeasure.multipleRestMeasures !== undefined && multipleRestNumber !== currentMeasure.multipleRestMeasures) {
-                  // different multi-rest values in same measure for different staffs
-                  currentMeasure.multipleRestMeasures = 0; // for now, ignore multirest here. TODO: take minimum
-                  // currentMeasure.multipleRestMeasuresPerStaff.setValue(this.currentStaff?.Id, multipleRestNumber);
-                  //   issue: currentStaff can be undefined for first measure
-                } else {
-                  currentMeasure.multipleRestMeasures = multipleRestNumber;
-                  this.currentMultirestStartMeasure = currentMeasure;
-                  this.followingMultirestMeasures = multipleRestNumber + 1; // will be decremented at the start of the loop
-                }
-              } catch (e) {
-                console.log("multirest parse error: " + e);
-              }
-            }
-          }
-
-        } else if (xmlNode.name === "forward") {
+        }
+        else if (xmlNode.name === "forward") {
           const forFraction: number = parseInt(xmlNode.element("duration").value, 10);
           currentFraction.Add(new Fraction(forFraction, 4 * this.divisions));
-        } else if (xmlNode.name === "backup") {
+        }
+        else if (xmlNode.name === "backup") {
+          console.log("backup node gevonden:");
```

Comment:
@sschmidTU I removed the dutch comments and console.logs that are not longer needed. I also changed demo/index.js back to it's original state.
I pushed it to my branch. 

Do I need to do another pull request?
PS Thanks for your feedback. This is my first time contributing to a github project, so please indulge me a bit :)

Reply:
No problem, thanks! No, another pull request is not necessary, just maybe some changes (on the same branch, which will update the pull request).
However, I would like to merge #1341 first, because the formatting there should be corrected first and then merged into this PR.